pkg/events/types.go

```diff
@@ -24,5 +24,9 @@ const (
 	// ReasonPortResolutionError is a reason for an event to be emitted.
 	ReasonPortResolutionError = "PortResolutionError"
 
+	// ReasonNoPrivateIPError is a reason for an event to be emitted.
 	ReasonNoPrivateIPError = "NoPrivateIP"
+
+	// ReasonRedirectwithNoTLS is a reason for an event to be emitted.
+	ReasonRedirectWithNoTLS = "RedirectWithNoTLS"
```

Comment:
This does not seem to be used anywhere... perhaps... just yet...

Reply:
`ReasonRedirectWithNoTLS` is used in pruneRedirectWithNoTLS function when creating the event.